gnovm/pkg/gnolang/preprocess.go

```diff
@@ -3051,6 +3063,20 @@ func checkIntegerKind(xt Type) {
 	}
 }
 
+func checkAssignmentMismatch(cd *ValueDecl) {
+	var (
+		numNames  = len(cd.NameExprs)
+		numValues = len(cd.Values)
+	)
+	if numValues <= 0 || numValues == numNames {
```

Comment:
there's no chance numValues <= 0, right?

Reply:
Yes right we only need this `numValues == 0`

Reply:
edited here: 89d1130af546002437b6ce2daa47553573bd30c8

Reply:
`numValues == 0` when RHS is empty? that won't parse at all.